docs/guides/server/pinned-guides

```diff
@@ -1,3 +1,4 @@
+directory-structure
```

Comment:
I would not place it so high. The first think user usually needs is configuration and admin user. I'd place it between `bootstrap-admin-recovery` and `containers`. WDYT?

Reply:
Seems fine.